cob_hardware_config/CMakeLists.txt

```diff
@@ -25,3 +25,7 @@ install(DIRECTORY robots test
 install(FILES upload_robot.launch
   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 )
+
+install(FILES xacrodisplay.launch
```

Comment:
better name `display_robot.launch`?
(you could also combine the two `FILES` install tags...